src/js/components/OptionalPortsAndServiceDiscoveryComponent.jsx

```diff
@@ -246,7 +257,11 @@ var OptionalPortsAndServiceDiscoveryComponent = React.createClass({
   },
 
   render: function () {
-    if (isTooComplexStructure(this.props.fields)) {
+    var hasPlaceIdSet = PluginComponentStore.hasComponentAtPlaceId(
+      PluginMountPoints.OPTIONAL_PORTS_AND_SERVICE_DISCOVERY
+    );
+
+    if (!hasPlaceIdSet && isTooComplexStructure(this.props.fields)) {
```

Comment:
Yeah that's clear – I wonder if we should let the plugin decide if it can handle the stuff or not...